docs/csharp/language-reference/operators/lambda-operator.md

```diff
@@ -93,8 +100,35 @@ static void Main(string[] args)
     // nine  
 }  
 ```  
-  
+## Expression body definition
+
+An expression body definition provides a member's implementation in a highly condensed, readable form. It has the following general syntax:
+
+```csharp
+member => expression;
+```
+where *expression* is a valid expression.
```

Comment:
I might clarify this:

> where *expression* is a valid expression, with the restriction that a *statement expression* can if and only if the member's return type is `void`, or the member is a constructor, or a finalizer.

Reply:
Thanks, @BillWagner. I noted the restrictions on statement expressions. I'll merge when the build completes.